### ISSUE TITLE
Handling small UX/UI issues in OSD projects

### ISSUE THREAD

**Issue by Xaviju** [opensourcedesign/organization#48]
Hi!
Is there a way to solve small UX/UI questions for Open Source projects? 
Questions that are not a _job_, and can be probably responded pretty straight-forward.

Let me explain the context:
- In Taiga some time ago we had [an issue](https://github.com/taigaio/taiga-front/issues/1155) and we had not enough knowledge to answer it. Sometimes we do not have the time, the team or the knowledge. I asked on the IRC en within minutes @gillisig helped us (thanks again!)
- This happened again yesterday (and not in a good way) with this [issue about mistery meat](https://github.com/taigaio/taiga-front/issues/949). Again, we would love more OSD contributors to get more opinions.

I think it would be nice to have an open UI clinic with short Q&A that solve specific issues that can benefit many other projects. What do you think, would this be helpful?

Once the issue is solved it could be asked to contribute to the Open Collective :)

**Comment by belenbarrospena** [comment]
Is this the kind of thing that is suitable for a forum or mailing list? We could have an Open Source Design Stack Exchange (or open source equivalent), or create a mailing list ... although that would mean our communications growing tentacles ;)

**Comment by eppfel** [comment]
I actually thought about this on my way home from FOSDEM. A quick, not-final solution (because it restricts to github) would be a github team lets say @opensourcedesign/taskforce or something similar.

OSD members could add themselves, if they are up to the task and we advertise projects to directly mention this team in on github.

Of course there is the big question, how would join such a taskforce? And how to advertise this. On the other hand this could help OSD spread the word as well.

**Comment by simonv3** [comment]
I think @eppfel's solution is pretty elegant for the time being.

My main concern is that this would cause confusion between what constitutes a job and a "quick feedback" item. Where does "UX feedback" fall into this? What about "what do you think of this logo design"? What's the line? 

Ideally we'd have a sustainable space where people could pop in and ask questions like that. The issue is that IRC is too slow to do so right now.

**Comment by bnvk** [comment]
> I think @eppfel's solution is pretty elegant for the time being.

Me too :)

> My main concern is that this would cause confusion between what
> constitutes a job and a "quick feedback" item. Where does "UX feedback"
> fall into this? What about "what do you think of this logo design"? What's
> the line? 

That's a good point. Hrm. Perhaps a couple groups:

- Task Force
- Design Feedback

**Comment by simonv3** [comment]
I meant more along the lines of - what's the difference between our job board and a task force?

**Comment by Xaviju** [comment]
I like the @eppfel idea a lot!
If I were able to just mention a repo in an issue to make a kind of _bat call_ to OSD designers to solve a fast question in an OS project that would be **awesome**!
It would be nice to keep a registry of solved issues, it could probably help a lot of projects.

@simonv3 i think of the job board as a place where projects ask for help: with specifications and expecting an output, or a paid job. Of course, it could be confusing, but if a project uses the _OSD bat call_ to solve a job, we should point them to the job form or repo. What do you think? 

I would be happy to edit the website to ensure that this difference is well understood by OS projects.

**Comment by simonv3** [comment]
Does mentioning repos work that way? That _would_ be pretty cool. 

I think if we make sure to not take on jobs with the bat call than this is a fair way to deal with it! I say let's go for it and see what happens.

**Comment by Xaviju** [comment]
A user can _@mention_ issues from another repos: https://guides.github.com/features/issues/ 
> Issue in another repository? Just include the repository before the name like `opensourcedesign/osdcall#1`

My final suggestion is:
1. Create a repo with a fancy name with a single open issue.
2. Lock conversation to avoid chatting on this issue.
3. Add a comment in the issue or create a CONTRIBUTING explaining how it works.
4. In the website jobs section, add an explanation how the _OSD bat call_ works.
5. Link this help file from the home section.

What do you think? I volunteer to do it.

**Comment by simonv3** [comment]
FWIW this got the 👍  in the meeting.

**Comment by eppfel** [comment]
I just realized team mentions are not possible outside of an organization.
<img width="455" alt="bildschirmfoto 2017-02-15 um 22 41 21" src="https://cloud.githubusercontent.com/assets/1899308/22996818/66902d8e-f3d0-11e6-8f89-51de44856c6c.png">

So, @Xaviju's solution is the only one we have.

---

What should the ~~team~~ repo be called? Maybe in the future there could be the need for multiple ~~teams~~ repos.

- uxhelp
- artdirection
- …

For now something funny, so people would remember it and did not have to look it up each time would be best:

Maybe something like: 
- opensourcedesign/rescue
- opensourcedesign/firstaid
- opensourcedesign/emergency

**Comment by jancborchardt** [comment]
Yup – team mentions are only possible inside of an organization. ;)

I’d definitely say we should just handle it directly via Github issues like we do everything else. A separate message board would make no sense and separate communications.

Maybe also in the interest of _not_ again creating more repositories which will make it more confusing, it might be best to just have a **tag** in the jobs repository called »help needed« for example.
Those would not make it into the actual job board – or actually could later, when we have better means of filtering. :)
Cause – why not get others involved in small questions like this?

**Comment by Xaviju** [comment]
@jancborchardt I'm not sure of the exact implementation. 
- I agree with you that keeping related things together is better and simpler. A new repo could be confusing and we are not sure if anyone (other than me) would use it.
- In the other hand I am afraid of making the jobs board too clumsy (its a bit confusing right now) and over the time even unusable. __Jobs__ name was also not very appropriated when it was more a _question_. A __batcall__ is much faster than adding an issue.

If we think that the _help needed_ github issues are useful enough I definitely would not add it in the job board on the website. Maybe we should add some tip so projects with UX/UI questions are not afraid of adding a job because is not as __important__ as a _real job_.

btw thanks for the comment in the Taiga issue ;)
